2.3.3/bullseye/include/pip.conf

```diff
@@ -1,3 +1,4 @@
 [global]
+index-url = https://pip.astronomer.io/v2/
 extra-index-url = https://pip.astronomer.io/simple/
```

Comment:
```suggestion
```

We should get rid of this one while we are at it.

Reply:
We still have version check and fab security manager at /simple.

Reply:
We should republish them on v2 then (or install via url). (extra-index-url is a security concern, part of the driver for this change)